asset_dashboard/models.py

```diff
@@ -37,29 +37,41 @@ class Project(models.Model):
     commissioner_districts = models.ManyToManyField('CommissionerDistrict', blank=True)
     zones = models.ManyToManyField('Zone', blank=True)
 
-    PHASE_CHOICES = [
+    def __str__(self):
+        return self.name
+
+
+class Task(models.Model):
+    """
+    A sub-project unit of work. Projects without defined tasks are assigned a
+    default Task of type "implementation".
+    """
+
+    # TODO: Identify better choice list (e.g., what's the difference between
+    # Phase 3 Construction and Construction?)
+    TASK_TYPE_CHOICES = [
         ('phase_1', 'Phase 1'),
         ('phase_2', 'Phase 2'),
         ('phase_3_engineering', 'Phase 3 Engineering'),
         ('phase_3_construction', 'Phase 3 Construction'),
         ('feasibility', 'Feasibility'),
         ('design', 'Design'),
-        ('construction', 'Construction')
+        ('construction', 'Construction'),
+        ('implementation', 'Implementation'),
     ]
```

Comment:
also need a status field

[done, unscheduled, in-progress]

Reply:
Added.